File: app/src/test/java/org/worshipsongs/registry/DummyFragment2.java
package org.worshipsongs.registry;

import android.os.Bundle;
import androidx.core.app.Fragment;

import org.worshipsongs.listener.SongContentViewListener;

/**
 * Author : Madasamy
 * Version : 3.x.x
 */

public class DummyFragment2 extends Fragment implements ITabFragment
{
    @Override
    public int defaultSortOrder()
    {
        return 2;
    }

    @Override
    public String getTitle()
    {
        return "song_books";
    }

    @Override
    public boolean checked()
    {
        return false;
    }

    @Override
    public void setListenerAndBundle(SongContentViewListener songContentViewListener, Bundle bundle)
    {

    }
}

File: app/src/test/kotlin/org/worshipsongs/registry/DummyFragment2.kt
package org.worshipsongs.registry

import android.os.Bundle
import androidx.fragment.app.Fragment
import org.worshipsongs.listener.SongContentViewListener

/**
 * Author : Madasamy
 * Version : 3.x.x
 */
class DummyFragment2 : Fragment(), ITabFragment {
    override fun defaultSortOrder(): Int {
        return 2
    }

    override val title: String
        get() = "song_books"

    override fun checked(): Boolean {
        return false
    }

    override fun setListenerAndBundle(
        songContentViewListener: SongContentViewListener?,
        bundle: Bundle
    ) {
    }
}